codecov.yml

```diff
@@ -0,0 +1,9 @@
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+
+  status:
+    project: no
+    patch: yes
+    changes: no
```

Comment:
Let's use `.codecov.yml` as the name, instead.
That's also valid, see https://docs.codecov.io/docs/codecov-yaml#section-repository-yaml
That'd be more consistent with our `.gitignore` and `.travis.yml`, and would keep me happy! :smile:. I probably don't mind about the config file existing at that point.